src/test/ui/terminal-width/whitespace-trimming-2.stderr

```diff
@@ -0,0 +1,14 @@
+error[E0308]: mismatched types
+  --> $DIR/whitespace-trimming-2.rs:4:187
+   |
+LL | ...-> usize {
+   |       ----- expected `usize` because of return type
+LL | ...                                                                                                                                                                                 ()
```

Comment:
I think this could also benefit from more eager whitespace eating

Reply:
This test in particular _can't_ eat more without changing the shape of the code. The trimming uses bounding boxes, and what you see here is the smallest bounding box possible that points to both the return type and the `()` being returned by the tail expression. I personally prefer this behavior (at least to start with) to using different left side trimming on different lines.